Inseok_45/routes/userRouter.js

```diff
@@ -4,6 +4,7 @@ const userController = require('../controllers/userController');
 const router = express.Router();
 
 router.post('/signup', userController.signUp);
+router.post('/login', userController.getUserById);
```

Comment:
```suggestion
router.post('/login', userController.login);
```
컨트롤러 함수 이름 명확히 해주세요!